R/bfast01.R

```diff
@@ -8,14 +8,11 @@
 #' steps:
 #' 
 #' 1. The data is preprocessed with bfastpp using the arguments
-#' order/lag/slag/na.action/stl/sbins.
-#' 
+#' `order`/`lag`/`slag`/`na.action`/`stl`/`sbins`.
 #' 2. A linear model with the given formula is fitted. By default a suitable
 #' formula is guessed based on the preprocessing parameters.
-#' 
 #' 3. The model with 1 breakpoint is estimated as well where the breakpoint is
 #' chosen to minimize the segmented residual sum of squares.
-#' 
 #' 4. A sequence of tests the null hypothesis of zero breaks is performed. Each
```

Comment:
For me it's difficult to understand what this line means. It feels like there might be a grammatical error, but I don't know exactly what the author wanted. Feel free to ignore it, as it is an issue that is actually not in the scope of this pull request.